tools/ports/binaryen.py

```diff
@@ -1,6 +1,6 @@
 import os, shutil, logging
 
-TAG = 'version_42'
+TAG = 'version_43'
```

Comment:
Is this change intentional to coincide with this PR?

Reply:
Yeah, it depending on some fixes there. Meanwhile time has passed and a later version has been updated anyhow ;) So this will vanish in the merge.